presto-main/src/test/java/com/facebook/presto/metadata/TestPolymorphicScalarFunction.java

```diff
@@ -89,42 +89,6 @@ public void testSelectsMethodBasedOnReturnType()
         assertEquals(functionImplementation.getMethodHandle().invoke(INPUT_SLICE), VARCHAR_TO_BIGINT_RETURN_VALUE);
     }
 
-    @Test
-    public void testSelectsFirstMethodBasedOnPredicate()
-            throws Throwable
-    {
-        SqlScalarFunction function = SqlScalarFunction.builder(TestMethods.class)
-                .signature(SIGNATURE)
-                .deterministic(true)
-                .implementation(b -> b
-                        .methods("varcharToBigint")
-                        .withPredicate(context -> true))
-                .implementation(b -> b.methods("varcharToBigintReturnExtraParameter"))
```

Comment:
are those static test methods still used?

Reply:
Yes, these are all still used. Thank you for the review.